components/core/tests/test-encoding_methods.cpp

```diff
@@ -398,6 +406,13 @@ TEMPLATE_TEST_CASE("Encoding floats", "[ffi][encode-float]", eight_byte_encoded_
 
     value = "1.0L";
     REQUIRE(!encode_float_string(value, encoded_var));
+
+    if constexpr (std::is_same_v<TestType, eight_byte_encoded_variable_t>) {
```

Comment:
We can place the test case for INT64_MAX next to INT64_MIN. In this way the comments on the difference between these two can be more understandable